docs/statusreport-controller.md

```diff
@@ -3,113 +3,116 @@
 ```mermaid
 %%{init: {'theme':'forest'}}%%
 flowchart TD
-  %% Defining the styles
+%% Defining the styles
     classDef Amber fill:#FFDEAD;
 
-  predicate((PREDICATE: <br>Snapshot has annotation <br>test.appstudio.openshift.io/status <br>changed AND <br> it's not restored from backup))
+    predicate((PREDICATE: <br>Snapshot has annotation <br>test.appstudio.openshift.io/status <br>changed AND <br> it's not restored from backup))
 
 %%%%%%%%%%%%%%%%%%%%%%% Drawing EnsureSnapshotFinishedAllTests() function
 
 %% Node definitions
-  
-  get_required_scenarios(Get all required <br> IntegrationTestScenarios)
-  parse_snapshot_status(Parse the Snapshot's <br> status annotation)
-  check_finished_tests{Did Snapshot <br> finish all required <br> integration tests?}
-  check_supersede{Does Snapshot need <br> to be superseded <br> with a composite Snapshot?}
-  check_passed_tests{Did Snapshot <br> pass all required <br> integration tests?}
-  create_snapshot(Create composite Snapshot)
-  update_status(Update Snapshot status accordingly)
-  continue_processing_tests(Controller continues processing)
 
+    get_required_scenarios(Get all required <br> IntegrationTestScenarios)
```

Comment:
Can we reduce diff here? why is there change in indentation?

Reply:
fixed, must have been an editor thing? not sure what happened here.